internal/fullnode/labels.go

```diff
@@ -10,19 +10,25 @@ import (
 
 const (
 	networkLabel = "cosmos.strange.love/network"
+	typeLabel    = "cosmos.strange.love/type"
 )
 
 // kv is a list of extra kv pairs to add to the labels. Must be even.
 func defaultLabels(crd *cosmosv1.CosmosFullNode, kvPairs ...string) map[string]string {
 	if len(kvPairs)%2 != 0 {
 		panic(errors.New("key/value pairs must be even"))
 	}
+	nodeType := "FullNode"
```

Comment:
nit: Safer to use the constant already present in api/v1/cosmosfullnode_types.go instead of duplicating it here.

i.e.

```go
type FullNodeType string

const (
	FullNode FullNodeType = "FullNode"
	Sentry   FullNodeType = "Sentry"
)
```

Reply:
sure